lib/ansible/modules/cloud/vmware/vmware_guest.py

```diff
@@ -235,10 +235,19 @@
     default: '300'
     type: int
     version_added: '2.10'
+  wait_for_customization_timeout:
+    description:
+    - Define a timeout (in seconds) for the wait_for_customization parameter.
+    - Be careful when setting this value since the time guest customization took may differ among guest OSes.
+    default: '3600'
```

Comment:
3600 still looks long wait. Can we print message to user that customization takes longer. Just a thought not sure if it is possible to print from Ansible module. @Akasurde , is it possible to do this?

Reply:
It's just the default value, so make it long enough to make sure. If user wants to set it's own value then that's "wait_for_customization_timeout" parameter does.
The module just wait for expected events in vCenter, if no event found after the timeout, then it will fail. I think it can not find out the progress of customization in guest. Thanks.

Reply:
We can use ``self.module.warn``  to print warnings while playbook is running.

Reply:
Do you mean after every sleep(10s) it uses 'self.module.warn' to print 'still waiting for event' info?

Reply:
I had a try to use `self.module.warn`, this warning message is printed when timeout reached at the end of the task, not printed in the middle, so I think it can not indicate user that customization is taking long. Thanks.